src/@types/smee-client/index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "smee-client" {
+  import EventSource = require('eventsource')
+
+  interface Options {
+      source?: string
+      target: string
+      logger?: Console
```

Comment:
On second thought, the standard `Console` will clash with bunyan `Logger` because their interfaces are not exactly the same.

I looked at [the source of smee](https://github.com/probot/smee/blob/caed5d5ca9e9f8e30ced5c75b2ed854ee63075f0/client/index.js) and they are using just `logger.info` and `logger.error`. This means any object that implements `Console.info` and `Console.error` will do.

```ts
  type Severity = 'info' | 'error'

  interface Options {
      source?: string
      target: string
      logger?: Pick<Console, Severity>
  }
```

Such a definition will work with both standard `Console` and bunyan `Logger`. 👍 I apologize for not having thought about it earlier.